vaadin-messages-flow-parent/vaadin-messages-flow/src/main/resources/META-INF/resources/frontend/messageListConnector.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2000-2019 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+(function () {
+    const tryCatchWrapper = function (callback) {
+        return window.Vaadin.Flow.tryCatchWrapper(callback, 'Vaadin Message List', 'vaadin-messages');
+    };
+
+    window.Vaadin.Flow.messageListConnector = {
+        setItems: tryCatchWrapper(function (list, items, locale) {
+            const formatter = new Intl.DateTimeFormat(locale, { dateStyle: 'medium', timeStyle: 'short'});
```

Comment:
Have you tested what happens in IE11?

Reply:
Just did: both IE11 and Safari do not support `dateStyle` and `timeStyle`, but using
```js
new Intl.DateTimeFormat(locale, {
    year: 'numeric', month: 'short', day: 'numeric',
    hour: 'numeric', minute: 'numeric'
});
```
gives the same result in Chrome/Firefox/Safari while in IE11 it's slightly different but still localized (e.g. for Italian it formats `17/feb/2021` while others `17 feb 2021`).

Reply:
Good. :+1: Support for IE11 doesn't mean it needs to be perfectly consistent with other browsers, so that's good enough.